tests/test_main.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from app.main import battle
```

Comment:
Delete this change in your code, it blocks tests from running. You are not supposed to edit this file.